src/Parse.java

```diff
@@ -52,9 +63,33 @@ public Parse(String filename, boolean parseNPrint) {
         }
         return payBytes;
     }
+
+    public static boolean hasCmdArgsYamlFlag (String args[]) {
+	return args.length >= 1 && args[0].equals("-y");
+    }
+
+    public static String readYamlPathArgs (String args[]) {
+	if (hasCmdArgsYamlFlag(args)) {
+	    return args[1];
+	} else {
+	    return "..//itch5.yaml";
+	}
+    }
+
+    public static String readItchPathArgs (String args[]) {
+	int argsPos = hasCmdArgsYamlFlag(args) ? 2 : 0;
+	if (args.length > argsPos) {
+	    return args[argsPos];
+	} else {
+	    return "";
+	}
+    }
```

Comment:
@Mister-Meeseeks one liner ternary. I hate those while reading code this is where it'll look clean.